src/pages/_app.tsx

```diff
@@ -23,10 +22,13 @@ const client = new QueryClient()
 function LlamaAIWelcomeWrapper() {
 	const [shown, setShown] = useLlamaAIWelcome()
 	const isClient = useIsClient()
-	const { hasFeature, loading } = useFeatureFlagsContext()
+
 	const { subscription } = useSubscribe()
 	const router = useRouter()
 	const [showModal, setShowModal] = useState(false)
+	const { user } = useAuthContext()
+
+	const hasFeatureLlamaAI = user?.flags?.llamaai ?? false
```

Comment:
@ReynardoEW is there any need in this flag & related code after llamaai launch?

Reply:
yeah i think we can remove that modal